.github/workflows/security-testing.yml

```
@@ -0,0 +1,45 @@
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ['typescript']
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Cache CodeQL databases
+        uses: actions/cache@v3
```

actionlint (1.7.11)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)